src/DIG/InputTextarea.jsx

```diff
@@ -0,0 +1,15 @@
+let { minHeight, ...forwardedProps } = props;
+
+return (
+  <Widget
+    src="near/widget/DIG.Input"
```

Comment:
@calebjacob maybe I missed that but shouldn't we use `REPL_ACCOUNT` be here:

```suggestion
    src="${REPL_ACCOUNT}/widget/DIG.Input"
```

Reply:
Fixed in this PR: https://github.com/near/near-discovery-components/pull/160